deploy.sh

```diff
@@ -24,11 +24,8 @@ push_helm_charts() {
 	fi
 }
 
-if [ "${TRAVIS_BRANCH}" == 'csi-v0.3' ]; then
-	export ENV_RBD_IMAGE_VERSION='v0.3-canary'
-	export ENV_CEPHFS_IMAGE_VERSION='v0.3-canary'
-elif [ "${TRAVIS_BRANCH}" == 'master' ]; then
-	export ENV_CSI_IMAGE_VERSION='canary'
+if [ "${TRAVIS_BRANCH}" == 'release-v1.2.0' ]; then
+	export ENV_CSI_IMAGE_VERSION='v1.2.0'
```

Comment:
Same comment as the one in Makefile here as well.

Reply:
yes will revert it back to 1.2-canary